docs/libraries/Aleph.im-Python-SDK/Aggregates/security_key.md

```diff
@@ -0,0 +1,82 @@
+Inside most Messages, there is an **"address"** field. This is the address for which the message applies (for which address to apply this aggregate, who posted that item…).</br>
```

Comment:
Should this be in the "Message types" section ?

Reply:
I wanted to give a little introduction before talking about the security key. That's why i put this